### ISSUE TITLE
warning *****coincident ****

### ISSUE THREAD

**Issue by ztggithub** [haeginh/POLY2TET#7]
"I built a model using Rhino 8. When each layer is separate, poly2tet works fine. However, when combined and exported, using poly2tet results in a warning, and the program won't stop."

**Comment by ztggithub** [comment]
I would like to know which 3D modeling software and version were used to create the MRCP OBJ file. Thank you.

**Comment by ztggithub** [comment]
I would like to know which 3D modeling software and version were used to create your MRCP OBJ file. Thank you

**Comment by ztggithub** [comment]
Will intersecting faces of different layers also report an error?

**Comment by haeginh** [comment]
Of course. Poly2Tet is intended for the models without any intersection: self-intersection by a single shell and intersections by different shells.
